pylsp/plugins/rope_completion.py

```diff
@@ -17,9 +17,10 @@ def pylsp_settings():
 
 
 def _resolve_completion(completion, data):
+    # pylint: disable=broad-except
     try:
         doc = data.get_doc()
-    except AttributeError:
+    except Exception:
```

Comment:
```suggestion
    except Exception as e:
        log.debug("Failed to resolve Rope completion: %s", e)
```

Reply:
Or maybe even at `log` level?

Reply:
I think at `debug` level is fine, but up to you.